client/src/components/CheckoutForm.js

```diff
@@ -10,6 +10,7 @@ function stripeCheckoutHandler(result) {
         fetch('/create-subscription', {
             method: 'POST',
             headers: { 'Content-Type': 'application/json' },
+            credentials: "include",
```

Comment:
@bugatti12345 Make sure to add this for all future fetch in order to send cookie to backend

Reply:
Oh just saw that now. Thank you for reminding